src/3p.js

```diff
@@ -109,18 +109,25 @@ function executeAfterWriteScript(win, fn) {
  * @param {string} src
  */
 export function validateSrcPrefix(prefix, src) {
-
+  const validPrefixes = [];
   if (!isArray(prefix)) {
     prefix = [prefix];
   }
-
-  for (const p of prefix) {
-    if (src.indexOf(p) === 0) {
+  const totalPrefixes = prefix.length;
+  if (src !== undefined) {
```

Comment:
src cannot be undefined according to the type. We should definitely fail in that case.